src/app/shared/content-with-optional-title/content-with-optional-title.component.html

```diff
@@ -0,0 +1,5 @@
+<div *ngIf="content" class="content-with-optional-title">
+    <h2 *ngIf="title">{{ title | translate }}</h2>
+    <p *ngIf="hasInnerHtml" [innerHtml]="content"></p>
```

Comment:
Maybe it would be better to use a `<div>` here. Otherwise, it will end up with nested HTML inside the `<p>`.